### PR TITLE
add redhat family support

### PR DESCRIPTION
**Pull request by aboe76**
more os support:
- centos
- redhat
- fedora

### CODE REVIEW

File: cert/map.jinja [hunk 1]
```diff
@@ -23,6 +23,39 @@
...
+        'key_dir': '/etc/pki/tls/private',
+        'pkgs': [ 'ca-certificates' ],
+        'cert_user': 'root',
+        'key_user': 'root',
+        'cert_group': 'root',
+        'key_group': 'root',
+        'cert_mode': '600',
+        'key_mode': '600',
+    },
+		'Fedora': {
```
Comment (iggy) [inline cert/map.jinja]: tabs

File: cert/map.jinja [hunk 2]
```diff
@@ -23,6 +23,39 @@
...
+        'key_dir': '/etc/pki/tls/private',
+        'pkgs': [ 'ca-certificates' ],
+        'cert_user': 'root',
+        'key_user': 'root',
+        'cert_group': 'root',
+        'key_group': 'root',
+        'cert_mode': '600',
+        'key_mode': '600',
+    },
+		'Redhat': {
```
Comment (iggy) [inline cert/map.jinja]: tabs

Comment (aboe76) [inline cert/map.jinja]: fixed with new commit

**Comment by aboe76** [comment]
closing this one because of #4

**Pull request closed without merging**